### ISSUE TITLE
Not able to find  current user I'd for sending messages in messagekit

### ISSUE THREAD

**Issue by reginaagst** [MessageKit/MessageKit#843]

**Comment by reginaagst** [comment]
In JSQMESSagesViewController it had a property called senderID. The senderId was a property of JSQMESSagesViewController that allowed the string identifier to uniquely identify the current user sending messages. My issue is I read the documentation's quick start guide for Messagesviewcontroller and I looked through the example project . I learned that to start using messageKit that I needed to subclass Messagesviewcontroller in my project and I realize that Messagesviewcontroller is not identical to JSQViewcontroller. My issue is I am having trouble finding a way to identify my current user in my app. My question is there another class in message kit that allows this functionality like JsqmessagesViewController or is the subclass Messagesviewcontroller sender identification functionality built in my user with out me having to set it in code?
